### PR TITLE
email-capture-corner

### PR DESCRIPTION
**Pull request by Samuel270**

### CODE REVIEW

File: Web Experiences/Blueprints/email-capture-corner/email-capture-corner.css [hunk 1]
```diff
@@ -1,53 +1,49 @@
-/*
```
Comment (bx-conor-oshea) [inline Web Experiences/Blueprints/email-capture-corner/email-capture-corner.css:1]: As in the Alert Bar review here also

File: Web Experiences/Blueprints/email-capture-corner/email-capture-corner.css [hunk 2]
```diff
@@ -1,53 +1,49 @@
...
 
-#bx-transition-card {
+
+/* experience container styling */
+#bx-{{ref}} #bx-transition-card {
     position: fixed;
     width: 435px;
     height: 325px;
-    [[ Position | enum(Left, Right)||{order: 1, values(left, right)}]]: 0;
+    [[ Position | enum(Left, Right)| Right|{ values(left, right), , group: General }]]: 5px;
```
Comment (bx-conor-oshea) [inline Web Experiences/Blueprints/email-capture-corner/email-capture-corner.css]: Is this double comma here correct?

File: Web Experiences/Blueprints/email-capture-corner/email-capture-corner.js [hunk 3]
```diff
@@ -1,47 +1,83 @@
...
         "channel": "WEB",
         "type": eventType,
         "pos": window._boxever_settings.pointOfSale,
         "browser_id": Boxever.getID(),
         "interactionID":"OOB_EXP",
 	    "interactionName": "CORNER_POPUP_EMAIL"
     };
-    Boxever.eventCreate(eventToSend, function(data) {}, 'json');
+    Boxever.eventCreate(eventToSend, (data)=>{}, 'json');
 }
```
Comment (bx-conor-oshea) [inline Web Experiences/Blueprints/email-capture-corner/email-capture-corner.js]: As in Side Bar review

File: Web Experiences/Blueprints/email-capture-corner/email-capture-corner.js [hunk 4]
```diff
@@ -1,47 +1,83 @@
...
         "language": "EN",
         "currency": "EUR",
         "page": "Home",
         "pos": "spinair.com",
         "browser_id": Boxever.getID(),
-        "email": document.getElementById("bx-email_input").value
+        "email":email
     };
-    Boxever.eventCreate(event, function(data) {}, 'json');
+    Boxever.eventCreate(event, (data)=>{}, 'json');
```
Comment (bx-conor-oshea) [inline Web Experiences/Blueprints/email-capture-corner/email-capture-corner.js]: This is incorrect here. Corrected Identity event:
```
let identityEvent = {
        "channel": "WEB",
        "type": "IDENTITY",
        "pos": window._boxever_settings.pointOfSale,
        "browser_id": Boxever.getID(),
        "email": email
    };
    Boxever.eventCreate(identityEvent, function(data) {}, 'json');

File: Web Experiences/Blueprints/email-capture-corner/email-capture-corner.css (hunk 1 above)
Comment (Samuel270) [reply Web Experiences/Blueprints/email-capture-corner/email-capture-corner.css:1]: Done

File: Web Experiences/Blueprints/email-capture-corner/email-capture-corner.css (hunk 2 above)
Comment (Samuel270) [reply Web Experiences/Blueprints/email-capture-corner/email-capture-corner.css]: was a typo, Thankls

File: Web Experiences/Blueprints/email-capture-corner/email-capture-corner.js (hunk 3 above)
Comment (Samuel270) [reply Web Experiences/Blueprints/email-capture-corner/email-capture-corner.js]: all standardized

File: Web Experiences/Blueprints/email-capture-corner/email-capture-corner.js (hunk 4 above)
Comment (Samuel270) [reply Web Experiences/Blueprints/email-capture-corner/email-capture-corner.js]: Done

**Review by bx-conor-oshea — APPROVED** [review]
Looks good. Only comments mentioned on other PRs. Also this branch is a fork, rather than an ordinary branch off the master here which would be preferred in future to keep things consistent.

**Pull request merged**